### PR TITLE
Added - Preview + Link if the data is too long. Removed - Asterisks from Example

### PR DESCRIPTION
**Pull request by Cyan101**
Added - Preview + Link if the data is too long. Removed - Asterisks from Example

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 DAH is Cards Against Humanity, integrated into Discord.
 
-![logo](http://imgur.com/MiqYOYD.png)
+![logo](http://i.imgur.com/2R6gSEV.png)
```
Comment (z64) [inline README.md]: In the future, please keep unrelated changes to separate PR's.

File: src/modules/commands/urban.rb [hunk 2]
```diff
@@ -19,9 +19,10 @@ def long?
         next "Couldn't find anything for `#{term}` 😕" unless word
 
         event.channel.send_embed do |e|
-          e.description = '**Too long to display! Visit the URL by clicking above.**' if word.long?
-          e.add_field name: 'Definition', value: word.text, inline: false unless word.long?
-          e.add_field name: 'Example', value: "*#{word.example}*", inline: false if word.example unless word.long?
+          e.add_field name: 'Definition',
+            value: word.long? ? "#{word.text[0..600].strip}... [(Read More)](#{word.url})" : word.text, inline: false
+          e.add_field name: 'Example',
+            value: "*#{word.long? ? word.example.gsub('*','')[0..600].strip + '...' : word.example.gsub('*','')}*", inline: false if word.example
```
Comment (z64) [inline src/modules/commands/urban.rb]: Both of these interpolations are now doing too much work, making this difficult to read (performing these operations has nothing to do with presenting the embed field). This should be abstracted into a helper method, perhaps similar to how I patched in `#long?`, but perhaps not, as this is a hack to begin with.

Beyond this, we should show as much of the definition as possible, not just 600 characters. We know exactly how long the markdown link will be, so you can come up with a way to replace exactly that amount off of the end.

If I were writing this, I would make the markdown link a template ;

```ruby
READ_MORE = ' ...(Read More)](url)'
```

and then generate the markdown replacement with the url like:

```ruby
url = READ_MORE.gsub 'url', word.url
text = word.text[0..-url.length].strip + url
```

which can then be suitably presented in the interpolation inside the embed.

**Review by z64 — CHANGES_REQUESTED** [review]

**Comment by Cyan101** [comment]
Commited some changes

File: src/modules/commands/urban.rb [hunk 3]
```diff
@@ -17,21 +21,33 @@ def long?
...
         next "Couldn't find anything for `#{term}` 😕" unless word
+        url = "... [(Read More)](#{word.url})"
+        word.example.delete!('*')
 
         event.channel.send_embed do |e|
-          e.description = '**Too long to display! Visit the URL by clicking above.**' if word.long?
-          e.add_field name: 'Definition', value: word.text, inline: false unless word.long?
-          e.add_field name: 'Example', value: "*#{word.example}*", inline: false if word.example unless word.long?
+          e.add_field name: 'Definition',
+                      value: word.long_text? ? truncate(word.text, url) : word.text, inline: false
```
Comment (z64) [inline src/modules/commands/urban.rb]: don't split the lines here, its inconsistent with the rest of the file. one line is fine

File: src/modules/commands/urban.rb [hunk 4]
```diff
@@ -17,21 +21,33 @@ def long?
...
           e.add_field name: "\u200B", value: "⬆️ `#{word.thumbs_up}` ⬇️ `#{word.thumbs_down}`", inline: false
           e.author = {
             icon_url: 'http://www.dimensionalbranding.com/userfiles/urban_dictionary.jpg',
             name: word.word,
             url: word.url
           }
-          e.footer = { text: "Author: #{word.author}"}
-          e.color = 5800090
+          e.footer = { text: "Author: #{word.author}" }
+          e.color = 5_800_090
```
Comment (z64) [inline src/modules/commands/urban.rb]: Revert this

**Review by z64 — CHANGES_REQUESTED** [review]

**Review by z64 — APPROVED** [review]

**Comment by Cyan101** [comment]
Good stuff :)

**Pull request merged**